### PR TITLE
Daemonset

### PR DESCRIPTION
**Pull request by Eoghan1232**
this needs to be rebased once the getPluginMode is merged

### CODE REVIEW

**Comment by davecremins** [comment]
I believe this needs a rebase now that the getPluginMode has been added?

File: e2e/artefacts/k8s/vsp-ds.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  template:
+    metadata:
+      labels:
+        name: vsp
+    spec:
+      hostNetwork: true
+      automountServiceAccountToken: false
+      containers:
+      - name: appcntr1
+        image: silpixa00400458:5000/intel-ipuplugin:latest
```
Comment (palade) [inline e2e/artefacts/k8s/vsp-ds.yaml:18]: Hardcoded variable that needs to be updated beyond demo.

File: e2e/artefacts/k8s/vsp-ds.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
+        command: [ "/usr/bin/ipuplugin" ]
+        args: [ "--bridgeType=linux", "--interface=enp0s1f0d3", "--ovsCliDir=/opt/p4/p4-cp-nws", "-v=debug"]
```
Comment (palade) [inline e2e/artefacts/k8s/vsp-ds.yaml:23]: enp0s1f0d3 is a hardcoded variable that needs to be updated beyond demo.

**Review by palade — APPROVED** [review]
There are a couple variable that should be updated after the demo, but aside from that this lgtm.

**Review by ahalimx86 — APPROVED** [review]

**Pull request merged**